.github/workflows/build-and-integration-tests.yml

```diff
@@ -103,29 +122,11 @@ jobs:
       if: ${{ matrix.target == 'x86_64-unknown-linux-gnu' && !startsWith(github.ref, 'refs/tags/v') }}
       run: sudo ./ci-scripts/install-deps.sh
 
-    - name: Build release
-      uses: actions-rs/cargo@v1
-      if: ${{ startsWith(github.ref, 'refs/tags/v') }}
-      with:
-        command: build
-        args: --target ${{ matrix.target }} --release ${{ matrix.release_build_args }}
-        use-cross: ${{ matrix.use-cross }}
-
-    - name: Build debug
-      uses: actions-rs/cargo@v1
-      if: ${{ ! startsWith(github.ref, 'refs/tags/v') }}
-      with:
-        command: build
-        args: --target ${{ matrix.target }} ${{ matrix.debug_build_args }}
-        use-cross: ${{ matrix.use-cross }}
-
-    - name: Copy and rename utility
-      if: ${{ startsWith(github.ref, 'refs/tags/v') }}
-      run: cp target/${{ matrix.target }}/release/${{ matrix.output }} ${{ matrix.output }}
+    - name: Build
+      run: ${{ env.CTOOL }} build --target ${{ matrix.target }} --profile ${{ env.CPROFILE }}
```

Comment:
Can we somehow keep the matrix.{release, debug}_build_args here?

Reply:
We can! I'd missed those